<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>322</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_right">
       <item>
        <widget class="QLabel" name="label_to_message_right">
         <property name="text">
          <string>Default .svg: flag with inner and outer trasnparency</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_left"/>
       </item>
       <item>
        <widget class="QLabel" name="label_bottom_message_left">
         <property name="text">
          <string>You can change the background color or choose another .svg file</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_change_color_left">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_open_file_left">
         <property name="text">
          <string>Open .svg file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_left">
       <item>
        <widget class="QLabel" name="label_top_message_right">
         <property name="text">
          <string>Default .svg: flag with inner transparency and white background</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_right"/>
       </item>
       <item>
        <widget class="QLabel" name="label_bottom_message_right">
         <property name="text">
          <string>You can change the background color or choose another .svg file</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_change_color_right">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_open_file_right">
         <property name="text">
          <string>Open .svg file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
